build.gradle

```diff
@@ -98,6 +106,14 @@ subprojects {
 }
 
 allprojects {
+    configurations {
```

Comment:
please add a a comment that this will be removed in the future when spring is updated.